python/tvm/tir/tensor_intrin/x86.py

```diff
@@ -67,8 +68,50 @@ def dot_product_16x4_u8i8i32_vnni(
         )
 
 
+mem_scope="global"
+@T.prim_func
+def dot_product_16x4_u8i8i32_avx512(a: T.handle, b: T.handle, c: T.handle,) -> None:
+    A = T.match_buffer(a, (4,), "uint8", offset_factor=1, scope=mem_scope)
+    B = T.match_buffer(b, (16, 4), "int8", offset_factor=1, scope=mem_scope)
+    C = T.match_buffer(c, (16,), "int32", offset_factor=1, scope=mem_scope)
```

Comment:
[cosmetic] Can be similar to the ```vnni``` part, as below ? https://github.com/apache/tvm/blob/209845fb910aac76f1388d868bb4fe4a46d9170f/python/tvm/tir/tensor_intrin/x86.py#L44-L49

Also, scope ```global``` is default anyway.

Reply:
Ok. I had some problems in this part of code and it was try to resolve them (I used hexagon example). But now the problems were resolved and it can be returned back. I believe it will be ok.

Reply:
done